include/dlr.h

```diff
@@ -60,8 +78,8 @@ int CreateDLRModel(DLRModelHandle* handle, const char* model_path, int dev_type,
  DLRGetLastError() to get the error message.
  */
 DLR_DLL
-int CreateDLRModelFromTFLite(DLRModelHandle* handle, const char* model_path,
-                             int threads, int use_nnapi);
+int CreateDLRModelFromTFLite(DLRModelHandle* handle, const char* model_path, int threads,
```

Comment:
Do we still need FromTFLite?

Reply:
im not changing it here, just ran clang-format per @trevor-m 's instructions